app/assets/stylesheets/govuk_frontend_toolkit/_typography.scss

```diff
@@ -3,6 +3,8 @@
 @import '_device-pixels.scss';
 @import '_url-helpers.scss';
 
+$text-indent: 50px;
```

Comment:
Agreed, except our typography file isn't high enough in the stack. Could just put it in `site.scss`?